src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -1205,51 +1204,49 @@ public void testCommandLineSinglePdf() throws IOException, TransformerException,
      * Test whether the command-line client can pick one of several entries from a bibtex file
      * @throws IOException
      * @throws TransformerException
-     *
      */
     @Test
-    @Ignore
     public void testCommandLineByKey() throws IOException, TransformerException {
-
         File tempBib = File.createTempFile("JabRef", ".bib");
         try (BufferedWriter fileWriter = Files.newBufferedWriter(tempBib.toPath(), StandardCharsets.UTF_8)) {
             fileWriter.write(t1BibtexString());
             fileWriter.write(t2BibtexString());
+        }
 
-            { // First try canh05
-                PrintStream oldOut = System.out;
-                try (ByteArrayOutputStream s = new ByteArrayOutputStream()) {
-                    System.setOut(new PrintStream(s));
-                    XMPUtilMain.main(new String[] {"canh05", tempBib.getAbsolutePath(), pdfFile.getAbsolutePath()});
-                } finally {
-                    System.setOut(oldOut);
-                }
-
-                // PDF should be annotated:
-                List<BibEntry> l = XMPUtil.readXMP(pdfFile, xmpPreferences);
-                Assert.assertEquals(1, l.size());
-                assertEqualsBibtexEntry(t1BibtexEntry(), l.get(0));
-            }
-            // Now try OezbekC06
-            try (ByteArrayOutputStream s = new ByteArrayOutputStream()) {
-                PrintStream oldOut = System.out;
-                System.setOut(new PrintStream(s));
-                try {
-                    XMPUtilMain.main(new String[] {"OezbekC06", tempBib.getAbsolutePath(), pdfFile.getAbsolutePath()});
-                } finally {
-                    System.setOut(oldOut);
-                }
-            }
+        PrintStream sysOut = System.out;
 
-            // PDF should be annotated:
-            List<BibEntry> l = XMPUtil.readXMP(pdfFile, xmpPreferences);
-            Assert.assertEquals(1, l.size());
-            assertEqualsBibtexEntry(t2BibtexEntry(), l.get(0));
+        // First try canh05
+        try (ByteArrayOutputStream s = new ByteArrayOutputStream()) {
+            System.setOut(new PrintStream(s));
+            XMPUtilMain.main(new String[]{"canh05", tempBib.getAbsolutePath(), pdfFile.getAbsolutePath()});
         } finally {
-            if (!tempBib.delete()) {
-                System.err.println("Cannot delete temporary file");
+            System.setOut(sysOut);
+        }
+
+        // PDF should be annotated:
+        List<BibEntry> l = XMPUtil.readXMP(pdfFile, xmpPreferences);
+        Assert.assertEquals(1, l.size());
+        assertEqualsBibtexEntry(t1BibtexEntry(), l.get(0));
```

Comment:
I do not like to assert the size of a list, because the result would simply say that the count mismatches, not which elements are in the list. Why not use the assertEqualsBibtexEntryList method if there is some which compares a list containing t1BibtexEntry with l?